openmdao/core/problem.py

```diff
@@ -935,6 +938,9 @@ def check_partials(self, out_stream=_DEFAULT_OUT_STREAM, includes=None, excludes
 
         model = self.model
 
+        if not model._use_derivatives:
```

Comment:
Probably should add this check to `check_totals` as well.

Reply:
Calling check_totals will call compute_totals, which raises an exception saying check_totals can't be called because derivs are turned off.